config/feature-policy.php

```diff
@@ -1,12 +1,17 @@
 <?php
 
 return [
+    // Feature-policy headers will only be added if this is set to true
+    'enabled' => env('FPH_ENABLED', true),
+
     /*
      * A policy will determine which Feature-Policy headers will be set.
      * A valid policy extends `Mazedlx\FeaturePolicy\Policies\Policy`
      */
     'policy' => Mazedlx\FeaturePolicy\Policies\Basic::class,
 
-    // Feature-policy headers will only be added if this is set to true
-    'enabled' => env('FPH_ENABLED', true),
+    'directives' => [
+        'proposal' => env('FPH_PROPOSAL_ENABLED', false),
```

Comment:
added the option to enable proposal and experimental permission policies. These could be implemented at a later date.